RFC/src/RFC-0140 Syncing and seeding.md

```diff
@@ -58,7 +58,7 @@ This document describes the process of Syncing, seeding, pruning and cut-through
 
 ### Syncing
 
-When a new node comes online, looses connection or encounters a fork that is longer than is longer than it can solve, it must enter syncing mode. This will allow it to recover its state to the newest up to date state. Sycning can be diveded into 2 [SynchronisationStrategy]s, complete sync and sync. Complete sync will involve that the node communicates with an archive node to get the complete history of every single block from genesis block. Sync will involve the node getting every block from its [puning horizon](pruninghorizon) to current head, as well as every block header from genesis block. 
+When a new node comes online, loses connection or encounters a chain re-organisation that is longer than it can tolerate, it must enter syncing mode. This will allow it to recover its state to the newest up to date state. Syncing can be divided into 2 [SynchronisationStrategy]s, complete sync and sync. Complete sync will involve that the node communicates with an archive node to get the complete history of every single block from genesis block. Sync will involve the node getting every block from its [pruning horizon](pruninghorizon) to current head, as well as every block header from genesis block. 
 
 #### Complete Sync
```

Comment:
Some typos:

- supply the "**complet**" history required to sync
- Choose the longest "**chain**" based on pow
- the "**pruninghorizon**" will just be infinite
- Validate blocks as if they "**where**" just mined and then received
- it should "**enclude**" the current total pow
- If the two chains "**dont**" match up
- pow has the "**responsiblity**" to ask 
- This will be "**hanled**" by the node 
- going "**backword untill**" a known block is found
- Only "**keepinng**" the current 
- MimbleWimble and mimblewimble vs. Mimblewimble
- only happens "**afterthe**" pruning horizon
- remove all spent outputs "**thats**" older than 
- in "**it's**" current stored [_Note: must be **its**_]